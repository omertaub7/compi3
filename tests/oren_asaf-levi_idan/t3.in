void explainWhileElse(int x) {
	int cnt = 0;
	while(cnt < x){ 
		printi(cnt);
		cnt = cnt + 1;
		if(cnt == 5) {
			break;
		}
	}
	else {
		print("shouldn't be printed");
	}
	
	while(cnt < x){ 
		printi(cnt);
		cnt = cnt + 1;
		if(cnt == 5) {
			break;
		}
	}
	else {
		print("should be printed");
	}
}

void main()
{
	int x = 7;
	explainWhileElse(x);
}