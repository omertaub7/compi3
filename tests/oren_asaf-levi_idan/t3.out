0
1
2
3
4
5
6
should be printed
